roles/openshift_hosted_metrics/tasks/install.yml

```diff
@@ -92,8 +92,8 @@
       HAWKULAR_METRICS_HOSTNAME={{ g_metrics_hostname }} \
       -v USE_PERSISTENT_STORAGE={{metrics_persistence | string | lower }} \
       -v DYNAMICALLY_PROVISION_STORAGE={{metrics_dynamic_vol | string | lower }} \
-      -v METRIC_DURATION={{ openshift.hosted.metrics.duration }} \
```

Comment:
this whole role should be able to go away actually... `openshift_metrics` is used